rust/src/krb/krb5.rs

```diff
@@ -364,6 +364,9 @@ pub unsafe extern "C" fn rs_krb5_probing_parser(_flow: *const Flow,
         input:*const u8, input_len: u32,
         _rdir: *mut u8) -> AppProto
 {
+    if input.is_null() {
+        return ALPROTO_UNKNOWN;
+    }
```

Comment:
I think the commit is worthy of a comment that we're checking from `FAILED` to `UNKNOWN`.

Reply:
added something

Reply:
Also https://redmine.openinfosecfoundation.org/issues/7019 down the rabbit hole